specification/network/resource-manager/Microsoft.Network/stable/2024-10-01/virtualNetworkGateway.json

```diff
@@ -4192,6 +4238,34 @@
       },
       "description": "List of virtual network gateway routes."
     },
+    "RadiusAuthServer": {
+      "type": "object",
+      "properties": {
+        "radiusServerAddress": {
+          "type": "string",
+          "description": "Radius server name"
+        },
+        "radiusServerSecret": {
+          "type": "string",
+          "description": "Radius server secret"
+        }
+      },
+      "description": "Gateway or VpnServerConfiguration Radius server with radius secret details"
+    },
+    "RadiusAuthServerListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
```

Comment:
> [] [](http://example.com/codeflow?start=29&length=3)

Is 'RadiusAuthServer' not usable as a unique key? I find that slightly surprising but am willing to go with it...

Reply:
yes RadiusAuthkey is unique. We are returning list of <RadiusAuthServer:RadiusSecret>